lib/core/constants/dashboard_constants.dart

```diff
@@ -0,0 +1,8 @@
+class DashboardConstants {
+  static const int calendarRange = 120;
+  static const double calendarItemWidth = 69.65;
```

Comment:
Same here, you don't need to specify the type, unless you want to and need to enforce e.g. 5 to be of type `double`.